src/consensus.rs

```diff
@@ -448,8 +448,15 @@ impl Consensus {
     }
 
     pub fn start(&mut self) -> anyhow::Result<()> {
-        let tick_period = Duration::from_millis(self.config.tick_period_ms);
+        // If this is the only peer in the cluster, tick Raft node a few times to instantly
+        // self-elect itself as Raft leader
+        if self.node.store().peer_count() == 1 {
```

Comment:
Does it make sense to make this check part of the loop?

I'm not sure if we might collect more peers before `has_ready() == true`.

Reply:
> Does it make sense to make this check part of the loop?

I don't think so.

> I'm not sure if we might collect more peers before has_ready() == true

We can't. We either collect them when bootstrapping new node/loading existing node from disk, or we have to apply some committed entry, which happens during `on_ready` call (under `has_ready` in the loop).